search-engines.lisp

```diff
@@ -622,6 +622,243 @@ REQUEST-ARGS is a list of args to pass to request function."
+                                  (:telugu "telugu")
+                                  (:thai "thai")
+                                  (:tigrinya "tigrinya")
+                                  (:turkce "turkce")
+                                  (:ukrainian "ukrainian")
+                                  (:urdu "urdu")
+                                  (:uzbek "uzbek")
+                                  (:vietnamese "vietnamese")
+                                  (:welsh "welsh")
+                                  (:xhosa "xhosa")
+                                  (:zulu "zulu")))
+  (web-date "with_date" ((:any "")
+                         (:day "d")
+                         (:week "w")
+                         (:month "m")
+                         (:year "y")))
+  (web-family-filter "qadf" ((:off "none")
+                             (:on "heavy")))
+  (web-region "qsr" ((:all "all")
+                     (:australia "en_AU")
+                     (:austria "de_AT")
+                     (:belarus "ru_BY")
+                     (:belgium-fr "fr_BE")
+                     (:belgium-nl "nl_BE")
+                     (:brazil "pt-BR_BR")
+                     (:bulgaria "bg_BG")
+                     (:canada "en_CA")
+                     (:canada-fr "en_FR")
+                     (:chile "es_CL")
+                     (:china "zh-CN_CN")
+                     (:denmark "da_DK")
+                     (:egypt "ar_EG")
+                     (:finland "fi_FI")
+                     (:france "fr_FR")
+                     (:germany "de_DE")
+                     (:greece "el_GR")
+                     (:honk-kong "zh-TW_HK")
+                     (:india "hi_IN")
+                     (:japan "ja_JP")
+                     (:korean "ko_KR")
+                     (:malaysia "ms_MY")
+                     (:malaysia-en "en_MY")
+                     (:netherlands "nl_NL")
+                     (:norway "no_NO")
+                     (:poland "pl_PL")
+                     (:portugal "pt_PT")
+                     (:romania "ro_RO")
+                     (:russia "ru_RU")
+                     (:south-africa "en_ZA")
+                     (:spain "es_ES")
+                     (:spain-ca "ca_ES")
+                     (:sweden "sv_SE")
+                     (:switzerland-de "de_CH")
+                     (:switzerland-fr "fr_CH")
+                     (:switzerland-it "it_CH")
+                     (:taiwan "zh-TW_TW")
+                     (:turkey "tr_TR")
+                     (:united-kingdom "en-GB_GB")
+                     (:united-states-en "en_US")
+                     (:united-states-es "es_US")))
+  (images-size "flimgsize" ((:any "")
+                            (:large "isz:l")
+                            (:medium "isz:m")
+                            (:large "isz:l")
+                            (:icon "isz:i")))
+  (images-size-predefined "image-size-select" ((:400x300 "isz:lt,islt:qsvgs")
```

Comment:
Is 400x300 the default size for image search? 

I should probably emphasize that in `define-search-engine` docstring, but the first value is automatically taken as default. So, you probably want to have `(:any "")` here, instead of `(:400x300 "isz:lt,islt:qsvgs")`.

Reply:
Whoops, yes!